### PR TITLE
[SD-2587] Get video from DC search and insert into SD

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: server/superdesk/aap_mm_datalayer.py [hunk 1]
```diff
@@ -115,15 +138,29 @@ def find_one_raw(self, resource, _id):
...
+            if doc['type'] == 'picture':
+                url = self._app.config['AAP_MM_SEARCH_URL'] + '/Assets/{}/Original/download'.format(_id)
+                mime_type = 'image/jpeg'
+            elif doc['type'] == 'video':
+                resolutions = self._get_resolutions(_id)
+                if any(v['Name'] == 'Ipod' for v in resolutions['Video']):
+                    url = self._app.config['AAP_MM_SEARCH_URL'] + '/Assets/{}/Ipod/download'.format(_id)
+                    mime_type = doc.get('renditions').get('original').get('mimetype')
+                else:
+                    raise NotImplementedError
```
Comment (amagdas) [inline server/superdesk/aap_mm_datalayer.py]: We should raise a specific error in this case I think.

**Comment by amagdas** [comment]
:+1: Except the `NotImplementedError`

**Comment by ioanpocol** [comment]
./superdesk/aap_mm_datalayer.py:34:34: E272 multiple spaces before keyword

Failing scenarios:
  features/aap_mm.feature:4  Can search multimedia

**Comment by ioanpocol** [comment]
:+1:  (except already mentioned issues)

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**